packages/paste-core/components/ai-chat-log/__tests__/AIChatLogger.spec.tsx

```diff
@@ -0,0 +1,35 @@
+import { render, screen } from "@testing-library/react";
+import * as React from "react";
+
+import { AIChatLogger, AIChatMessage, AIChatMessageBody } from "../src";
+import type { AIChat } from "../src/useAIChatLogger";
+
+const chats: AIChat[] = [
+  {
+    id: "uid1",
+    variant: "bot",
+    content: (
+      <AIChatMessage variant="bot">
```

Comment:
I'm wonder if API usage wise it would be possible to avoid having to pass the variant twice.

Reply:
Hey @henrieri, when using the hook you would have to define twice but if you're just using the logs you wouldn't need to. With the hook in the previous chat log it controlled the animation direction and we've kept the same logic for this.

This is currently the same way ChatLogger works and we want to maintain consistency between the components as they are so closely related. May make these components more intuitive for developers if they are already using ChatLogger.

@nkrantz we should discuss this though because it would make sense to have something like this in future.

Reply:
Same as above - agree but we want to mirror ChatLogger API here.

Reply:
Whoops - didn't see Kristian's comment. Agreed, would be nice to ticket this change across both packages.